geode-assembly/build.gradle

```diff
@@ -29,10 +27,17 @@ extraArchive {
   tests = false
 }
 
-// Gradle doesn't automatically remove the jar artifact even though we disabled it
-// this causes publishing to fail.  So we nuke all the disabled artifacts from all configurations.
+logger.info("Gradle doesn't automatically remove the jar artifact even though we disabled it")
+logger.info("this causes publishing to fail.  So we nuke all the disabled artifacts from all configurations.")
 configurations.all {
-  artifacts.removeAll artifacts.findAll { it instanceof ArchivePublishArtifact && !it.archiveTask.enabled }
+  def configName = it.name
+  artifacts.each {
+    logger.info("Pruning configName = " + configName + " name=" + it.name + " extension=" + it.extension + " classifier=" + it.classifier)
```

Comment:
Should this be `debug` level instead of `info`?

Reply:
As debug info it was lost in the noise, and logging instead of comments seems more-sane to me.